spec.html

```diff
@@ -52991,6 +53021,25 @@ <h1>Non-default behaviour in HostMakeJobCallback</h1>
       <h1>Non-default behaviour in HostEnsureCanAddPrivateElement</h1>
       <p>The HostEnsureCanAddPrivateElement abstract operation allows hosts which are web browsers to specify non-default behaviour.</p>
     </emu-annex>
+
+    <emu-annex id="sec-late-errors-for-function-call-assignment-targets">
+      <h1>Late Errors for Function Call Assignment Targets</h1>
+      <p>When a function call (<emu-xref href="#sec-function-calls"></emu-xref>) is used as an assignment target in non-strict code, instead of producing an early error, a *ReferenceError* exception may be thrown upon actually attempting the assignment.</p>
```

Comment:
```suggestion
      <p>When a function call (<emu-xref href="#sec-function-calls"></emu-xref>) is used as an assignment target in non-strict code, instead of producing an early error, a *ReferenceError* exception is thrown when evaluating the assignment.</p>
```